app/src/main/java/fr/free/nrw/commons/theme/NavigationBaseActivity.java

```diff
@@ -132,6 +133,17 @@ public boolean onNavigationItemSelected(@NonNull final MenuItem item) {
                     Toast.makeText(this, R.string.no_email_client, Toast.LENGTH_SHORT).show();
                 }
                 return true;
+            case R.id.action_rate:
+                drawerLayout.closeDrawer(navigationView);
+                final String appPackageName = getPackageName();
+                Toast.makeText(this, appPackageName, Toast.LENGTH_SHORT).show();
+                try {
```

Comment:
I would suggest extracting the rate us option logic to a different class(probably `Utils`). Later on we can have this option plugged in at multiple locations.

Reply:
Ok I will add a method in Utils.java.